{"cveId": "CVE-2020-27912", "cwe": ["CWE-787", "CWE-787"], "cvss": [{"baseScore": "7.8", "version": "3.1", "Attack Vector": "Low", "Attack Complexity": "Low", "Privileges Required": "None", "User Interaction": "Required", "Scope": "Unchaged", "Confidentiality Impact": "High", "Integrity Impact": "High", "Availability Impact": "High"}], "references": ["https://support.apple.com/en-us/HT211931", "https://support.apple.com/en-us/HT211935", "https://support.apple.com/en-us/HT211928", "https://support.apple.com/en-us/HT211929", "https://support.apple.com/en-us/HT211930", "https://support.apple.com/en-us/HT211933", "https://support.apple.com/kb/HT212011", "http://seclists.org/fulldisclosure/2020/Dec/32", "http://seclists.org/fulldisclosure/2020/Dec/26"], "description": ["An out-of-bounds write was addressed with improved input validation. This issue is fixed in macOS Big Sur 11.0.1, watchOS 7.1, iOS 14.2 and iPadOS 14.2, iCloud for Windows 11.5, tvOS 14.2, iTunes 12.11 for Windows. Processing a maliciously crafted image may lead to arbitrary code execution."], "published": "2020-12-08T21:15:13.420", "state": "PUBLIC", "vendorName": ["Apple"], "productName": ["watchOS", "iOS and iPadOS", "tvOS", "macOS", "macOS", "macOS"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}